opm/material/fluidmatrixinteractions/EclDefaultMaterial.hpp

```diff
@@ -439,26 +440,27 @@ class EclDefaultMaterial : public TraitsT
             //
             // Though be aware that from a physical perspective this is definitively
             // incorrect!
-            params.oilWaterParams().update(/*pcSw=*/  Sw, //1.0 - So, (Effect is significant vs benchmark.)
+            changed = changed || params.oilWaterParams().update(/*pcSw=*/  Sw, //1.0 - So, (Effect is significant vs benchmark.)
                                            /*krwSw=*/ 1.0 - So,
                                            /*krnSw=*/ 1.0 - So);
 
-            params.gasOilParams().update(/*pcSw=*/  1.0 - Swco - Sg,
+            changed = changed || params.gasOilParams().update(/*pcSw=*/  1.0 - Swco - Sg,
                                          /*krwSw=*/ 1.0 - Swco - Sg,
                                          /*krnSw=*/ 1.0 - Swco - Sg);
```

Comment:
Short-circuiting logical or here means that if `oilWaterParams().update()` returns `true` then we do not call `gasOilParams().update()`.  That's probably not what we want.